scipy/stats/_continuous_distns.py

```diff
@@ -781,12 +787,39 @@ def _pdf(self, x, c, d):
         # burr.pdf(x, c, d) = c * d * x**(-c-1) * (1+x**(-c))**(-d-1)
         return c * d * (x**(-c - 1.0)) * ((1 + x**(-c))**(-d - 1.0))
 
+    def _logpdf(self, x, c, d):
+        return np.log(c) + np.log(d) + (-c - 1.0) * np.log(x) \
+                + (-d - 1.0) * np.log1p(x**(-c))
+
     def _cdf(self, x, c, d):
         return (1 + x**(-c))**(-d)
 
+    def _logcdf(self, x, c, d):
+        return -d * np.log1p(x**(-c))
+
+    def _sf(self, x, c, d):
```

Comment:
This implementation of `_sf` is just `1 - cdf(x)`, so there is no need to include it.  That's what the default method does.  (There is a different formulation that is better at handling the right tail that might be worth implementing, but I'll experiment a bit and open a separate pull request if it looks worthwhile.)

Reply:
ok, removed `_sf`

Reply:
So removing `_sf` caused some errors in the unit tests for `fisk`. My solution was to reimplement `_sf` in `fisk` w/ `d=1.0`. The error message was too many arguments getting passed from burr to fisk, this came up on stats method unit test too.